### ISSUE TITLE
EE10 Unable to use Cookie attributes with `HttpServletResponse.addCookie(jakarta.servlet.http.Cookie)`

### ISSUE THREAD

**Issue by ekupcik** [eclipse/jetty.project#10227]
**Jetty version(s)**
12 Beta 4

**Description**
While trying to use Cookie.setAttribute("SameSite",...) without any success I figured out that HttpCookieFacade.getAttributes() always returns Collections.emptyMap() instead of delegating it to the underlying Cookie instance.

**Comment by joakime** [comment]
Can you show your code?
What technique you are using?

**Comment by joakime** [comment]
Never mind. I was able to replicate with ...

``` java
        HttpServlet testServlet = new HttpServlet()
        {
            @Override
            protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException
            {
                resp.setCharacterEncoding("utf-8");
                resp.setContentType("text/plain");

                Cookie cookie = new Cookie("key", "foo");
                cookie.setAttribute("SameSite", "Lax");
                resp.addCookie(cookie);

                resp.getWriter().println("hello");
            }
        };
```



PR incoming.

**Comment by joakime** [comment]
Opened PR #10228

**Comment by joakime** [comment]
Merged PR #10228
